kivy_ios/recipes/werkzeug/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 
 class WerkzeugRecipe(PythonRecipe):
-    version = "master"
+    version = "1.0.1"
```

Comment:
Makes an update to trigger the `rebuild_updated_recipes.py`. Also makes sure builds are reproducible as much as possible